### DISCUSSION TITLE
Website banner blocked third party cookie affect on PSAT cli tool?

### DISCUSSION THREAD

**Discussion by Joeyindigital** [GoogleChromeLabs/ps-analysis-tool#632] [Support Forum] [answered]
A senerio I was running the client site for audit 3pc, I found that with PSAT cli tool run a single url ( I used https://www.skoda.com.au/ in this case). The site has banner blocked third party cookie to allow user to accept cookie setting. 
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/166688925/89fa4c8a-9d2a-433b-8c98-3df5839b3ed5)
In this case, would it affect the result of running cookie scan via PSAT cli tool? 

I run it and the result is unbelievable low. My code is here: 
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/166688925/3eda54ab-f37b-4ba6-9d7e-63c7ff069d20)
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/166688925/bd6f5af0-ac9a-418c-b14b-09f016dcfc0b)

Any solution here if I prefer running 3PC audit via cli tool?

**Comment by milindmore22** [comment, marked as answer]
Hi @Joeyindigital,

We get your concern about GDPR banners which might affect your PSAT CLI analysis. We’re actively developing a feature to address it, please check GitHub issue #330.

This new functionality allows you to accept the GDPR banner directly within your CLI analysis.

Thanks for your patience!
